app/routes/session.py

```diff
@@ -128,9 +151,76 @@ def login() -> Response:
 
     cookie_session["language_code"] = metadata.language_code
 
+    if (dataset_id := metadata["sds_dataset_id"]) and ru_ref:
+        get_supplementary_data(
+            supplementary_data_url=SUPPLEMENTARY_DATA_URL,
+            dataset_id=dataset_id,
+            ru_ref=ru_ref,
+        )
+
     return redirect(url_for("questionnaire.get_questionnaire"))
 
 
+def get_supplementary_data(
+    supplementary_data_url: str, dataset_id: str, ru_ref: str
+) -> dict:
+    constructed_supplementary_data_url = (
+        f"{supplementary_data_url}?dataset_id={dataset_id}&unit_id={ru_ref}"
+    )
+
+    session = requests.Session()
+
+    retries = Retry(
+        total=SUPPLEMENTARY_DATA_REQUEST_MAX_RETRIES,
+        status_forcelist=SUPPLEMENTARY_DATA_REQUEST_RETRY_STATUS_CODES,
+    )  # Codes to retry according to Google Docs https://cloud.google.com/storage/docs/retry-strategy#client-libraries
+
+    # Type ignore: MyPy does not recognise BACKOFF_MAX however it is a property, albeit deprecated
+    retries.BACKOFF_MAX = SUPPLEMENTARY_DATA_REQUEST_MAX_BACKOFF  # type: ignore
```

Comment:
the library suggests using Default backoff max instead, is that not viable over the deprecated property? Appreciate this mirrors what `schema.py` does but may as well update this if its new (and you can)